bluebrain/deployment/environments/applications_libraries.yaml

```diff
@@ -30,14 +30,25 @@ spack:
       projections:
         all: '{name}/{version}'
         'omega-h+gmsh': '{name}/{version}-gmsh'
+
+  definitions:
+    - compilers: ['%gcc@11.2.0', '%gcc@9.4.0', '%clang']
```

Comment:
libraries compiled with `%gcc@9.4.0` are still required in our deployment? 🤔 (or, this is to help with gcc@11 issues that you saw with Caliper)